test/functional/test_framework/test_node.py

```diff
@@ -140,6 +151,23 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, bitcoind, bitcoin_cl
             AddressKeyPair('mzRe8QZMfGi58KyWCse2exxEFry2sfF2Y7', 'cPiRWE8KMjTRxH1MWkPerhfoHFn5iHPWVK5aPqjW8NxmdwenFinJ'),
     ]
 
+    def get_valgrind_version(self):
+        """Returns valgrind version (major, minor) in a tuple or None if not found"""
+        try:
+          rc = subprocess.call(['which', 'valgrind'],
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.PIPE)
+          if rc == 0:
+            version_regex = re.compile('[.bv]')
```

Comment:
1. Have you seen instances where the Valgrind version identifier contains `b` or `v`? :) If not, perhaps the simpler `version_info.split(".")` could do?

2. Check the indentation here so that it matches the four spaces used in the rest of this file :)

Reply:
I must admit that I wasn't sure abut valgrind's versioning. But I will adapt the code of course.
Will also update the indentations.